src/actions/users.js

```diff
@@ -151,11 +158,13 @@ export function loadMe() {
         const prefs = getMyPreferences()(dispatch, getState);
         const teams = getMyTeams()(dispatch, getState);
         const members = getMyTeamMembers()(dispatch, getState);
+        const teamUnreads = getMyTeamUnreads()(dispatch, getState);
 
         await me;
         await prefs;
         await teams;
         await members;
+        await teamUnreads;
```

Comment:
Do we need to wait on this or is it ok if it pops in a short time after loading? Just wondering if we can remove the `await`